iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -112,32 +112,6 @@ class ConversationListViewController: ViewController {
         self.collectionView.expandToSuperviewWidth()
         self.collectionView.top = self.headerVC.view.bottom
         self.collectionView.height = self.view.height - self.headerVC.view.bottom
-
-        // If we're in the write mode, adjust the position of the subviews to
-        // accomodate the text input, if necessary.
-        if self.state == .write {
-            self.setYOffsets()
-        }
-    }
-
-    /// Returns how much the collection view y position should  be adjusted to ensure that the text message input
-    /// and message drop zone don't overlap.
-    private func setYOffsets() {
```

Comment:
The collection view no longer moves based on text input. This was a huge performance boost.